vnext/Folly/Folly.vcxproj

```diff
@@ -254,7 +269,7 @@
       <AdditionalIncludeDirectories>$(ReactNativeWindowsDir)stubs;$(FollyDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>FOLLY_NO_CONFIG;NOMINMAX;_CRT_SECURE_NO_WARNINGS;WINAPI_PARTITION_APP;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <ForcedUsingFiles />
-      <DisableSpecificWarnings>4244;4251;4267;4293;4305;4800;4804;4310;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
4244 is to allow return type for a noreturn function .. We need that for glog stub : LOG(FATAL) << ""

Reply:
Are we able to fix the glog stub instead?

Reply:
Did you mean 4646 instead of 4244? The latter is about to be re-enabled as it's mandatory for SDL compliance.

Reply:
We no longer need this exception.